frontend/src/component/feature/FeatureView/FeatureMetrics/FeatureMetricsChart/createChartOptions.tsx

```diff
@@ -30,15 +41,38 @@ export const createChartOptions = (
                 padding: 10,
                 boxPadding: 5,
                 usePointStyle: true,
+                itemSort: (a, b) => {
+                    const order = ['Total requests', 'Exposed', 'Not exposed'];
+                    const aIndex = order.indexOf(a.dataset.label!);
+                    const bIndex = order.indexOf(b.dataset.label!);
+                    return aIndex - bIndex;
+                },
                 callbacks: {
+                    label: item => {
+                        return `${item.formattedValue} - ${item.dataset.label}`;
+                    },
+                    afterLabel: item => {
+                        const data = item.dataset.data[
+                            item.dataIndex
+                        ] as any as IPoint;
```

Comment:
the idiom is `as unknown as your_type`. However I'm always worried when we need to do this in production code (I use this idiom mostly for tests)